gulpfile.js

```diff
@@ -97,34 +104,23 @@ const bundleJsFiles = bundle => {
         .pipe(gulp.dest(finalDir));
 };
 
-const sassPipeline = () => {
-    return gulp.src(allBundleFiles('css', '.scss'), { base: buildDir, cwd: buildDir })
-        .pipe(gulpif(!production, sourcemaps.init()))
-        .pipe(sass({
-            sourceComments: !production,
-            outputStyle: production ? 'compressed' : 'nested'
-        }).on('error', handleError('SASS')))
-        .pipe(gulpif(!production, sourcemaps.write({
-            'includeContent': true
-        })))
-        .pipe(gulp.dest(finalDir));
-};
-
 /***********************
  * Start Tasks
  */
 
 /**
  * Delete the static_build directory and start fresh.
  */
-gulp.task('clean', () => {
```

Comment:
Note for reviewer: this task wasn't being called by anything previously. I'm guessing this might have been a mistake, but it could also have been intentional? 

I've now added this as the first task to both the default and build commands, so it will now always start afresh witha  clean folder each time it is run. Locally at least, this could help solve some issues with stray / redundant files being left around?